AppController/djinn.rb

```diff
@@ -4695,9 +4713,12 @@ def setup_appengine_application(app, state)
 
       if @app_info_map[app]['nginx'].nil?
         @app_info_map[app]['nginx'] = nginx_app_port
+      end
+      if @app_info_map[app]['nginx_https'].nil?
+        @app_info_map[app]['nginx_https'] = nginx_app_https
```

Comment:
I think we are covered by find_lowest_free_port and because this function is under lock. Do you think I miss a use case? btw I changed find free_port to check also partially allocated ports, just in case.